app/src/main/java/es/wolfi/app/passman/fragments/SettingsFragment.java

```diff
@@ -33,55 +34,56 @@
 import android.widget.ArrayAdapter;
 import android.widget.EditText;
 import android.widget.Spinner;
-import android.widget.Switch;
 import android.widget.TextView;
 
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
+import com.google.android.material.checkbox.MaterialCheckBox;
```

Comment:
I was wondering if this will work on devices with no google play services or other google addons.

Reply:
tried it with a plain Android 9 (without google apis) and it works fine
![grafik](https://user-images.githubusercontent.com/30630233/132358046-b64cbdfa-6433-417e-8221-bfb96da93ae9.png)